openshift/topics/advanced_concepts.adoc

```diff
@@ -284,7 +284,7 @@ route "sso" patched
 [[sso-connecting-to-an-external-database]]
 ==== Connecting to an external database
 
-{project_name} can be configured to connect to an external (to OpenShift cluster) database. In order to achieve this, you need to modify the `sso-{database name}` Endpoints object to point to the proper address. The procedure has been described in https://docs.okd.io/latest/dev_guide/expose_service/expose_internal_ip_service.html#manually-assign-ip-service[OpenShift manual].
+{project_name} can be configured to connect to an external (to OpenShift cluster) database. In order to achieve this, you need to modify the `sso-{database name}` Endpoints object to point to the proper address. The procedure has been described in https://docs.openshift.com/container-platform/4.2/networking/configuring-ingress-cluster-traffic/configuring-ingress-cluster-traffic-service-external-ip.html#nw-service-external-ip_configuring-ingress-cluster-traffic-service-external-ip[manually-assign-ip-address[OpenShift manual].
```

Comment:
@andymunro 

*1)* Please drop the `[manually-assign-ip-address` part from the change. Here's how it renders currently:

![ext_db_current](https://user-images.githubusercontent.com/8414918/74521134-39293500-4f19-11ea-95a7-f6b19b237ca8.png)

But the sentence should be `The procedure has been described in OpenShift manual`.

*2)*  Besides that, please change the fixed `4.2` value in the updated URL replacing with `latest`, at the corresponding place. IOW the new URL should be / becomes:

`https://docs.openshift.com/container-platform/latest/networking/configuring-ingress-cluster-traffic/configuring-ingress-cluster-traffic-service-external-ip.html#nw-service-external-ip_configuring-ingress-cluster-traffic-service-external-ip`

Compared with the currently proposed fixed form, the latter has the advantage it will be automatically rewritten to the latest available OCP version (in this case the above is transformed to the:

`https://docs.openshift.com/container-platform/4.3/networking/configuring-ingress-cluster-traffic/configuring-ingress-cluster-traffic-service-external-ip.html#nw-service-external-ip_configuring-ingress-cluster-traffic-service-external-ip`

one) when the link is visited, so it won't need to be updated WRT to next OCP docs version being available (till the page exists).